superset-frontend/tsconfig.json

```diff
@@ -20,7 +20,13 @@
     "sourceMap": true,
     "strictNullChecks": true,
     "suppressImplicitAnyIndexErrors": true,
-    "target": "es5"
+    "target": "esnext"
   },
-  "include": ["./src/**/*", "./spec/**/*"]
+  "include": [
+    "./src/**/*",
+    "./spec/**/*",
+    "./node_modules/*superset-ui*/**/src/**/*",
+    "./node_modules/*superset-ui*/**/types/**/*",
+    "./node_modules/*superset-ui*/**/node_modules/**/*.d.ts"
```

Comment:
Include any `superset-ui` related packages for TypeScript compiler. This solves most missing types errors.